provisioner/ansible/provisioner.go

```diff
@@ -485,7 +485,11 @@ func (p *Provisioner) createInventoryFile() error {
 	}
 
 	w := bufio.NewWriter(tf)
-	w.WriteString(host)
+	if _, err := w.WriteString(host); err != nil {
```

Comment:
😮  are you sure is ok to fail here?

Reply:
I'm not to be honest. I was hoping that it would be pointed out in the review 😄. My thought was that if it failed to write the header then the is probably some issue with writing to the underlying io.Writer (`w`) and all the other writes would fail. Maybe it is better to log the error here and let the call to w.Flush continue/fail as it currently does. 

I'll update to just log.

Reply:
Good idea!